.gitignore

```diff
@@ -1,3 +1,5 @@
 .anvil
 .DS_Store
 
+node_modules/
+package-lock.json
```

Comment:
Wouldn't adding these cause issues with the test `fixtures`? Or am I missing something?

Reply:
I think I'm missing something... don't we not want these to be committed for fixtures?

Reply:
Most of the time, but there are exceptions in the case that we want to test e.g. how the buildpack reacts when `node_modules` is checked in `test/fixtures/modules-checked-in` or assert how it behaves with and without a lockfile

Reply:
That makes sense, but wouldn't we want to make sure those stay out of git?